src/services/enum-utilities.ts

```diff
@@ -417,7 +419,10 @@ export default class EnumUtilities {
       case FilingTypes.CHANGE_OF_DIRECTORS: return FilingNames.CHANGE_OF_DIRECTORS
       case FilingTypes.CHANGE_OF_NAME: return FilingNames.CHANGE_OF_NAME
       case FilingTypes.CHANGE_OF_REGISTRATION: return FilingNames.CHANGE_OF_REGISTRATION
-      case FilingTypes.CONTINUATION_IN: return FilingNames.CONTINUATION_IN_APPLICATION
+      case FilingTypes.CONTINUATION_IN:
+        if (filingStatus === FilingStatus.APPROVED || filingStatus === FilingStatus.REJECTED) {
+          return FilingNames.CONTINUATION_IN_APPLICATION
+        } else { return FilingNames.CONTINUATION_AUTHORIZATION }
```

Comment:
I think there are other states for the application, like PAID, COMPLETED, CORRECTED and maybe others.

I think it would be better to check the states that are authorizations instead.

Reply:
I have the same question: It seems both Continuation in Authorization State and Application State can have the same statuses, like AWAITING_REVIEW, APPROVED or REJECTED. How do we differentiate them? Should we implement different statuses for the 2 stages?

Reply:
I think this is still not right. I will answer your question in your other comment.

Reply:
Thanks, will update now